lib/Echidna/Campaign.hs

```diff
@@ -108,20 +116,23 @@ runSymWorker
   -- ^ Initial corpus of transactions
   -> Maybe Text -- ^ Specified contract name
   -> m (WorkerStopReason, WorkerState)
-runSymWorker callback vm dict workerId initialCorpus name = do
+runSymWorker callback vm dict workerId _ name = do
```

Comment:
if this argument is not used, then let's delete it and adjust runWorker to not pass it

Reply:
I suspect these initial corpus will be used sooner than later, that´s why I prefer to keep it for now